docs/platforms/javascript/common/configuration/integrations/http.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Http
+description: "Capture spans & breadcrumbs for http requests. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.httpIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `httpIntegration` does two things:
+1. It captures spans for HTTP requests.
+2. It captures breadcrumbs for HTTP requests.
+
+
+## Options
+
+### `breadcrumbs`
+
+_Type: `boolean`_
+
+If set to false, no breadcrumbs will be captured.
+
+<PlatformSection notSupported={["javascript.fastify"]}>
+### `tracing`
+
+_Type: `boolean`_
+
+If set to true, spans will be captured for outgoing HTTP requests.
```

Comment:
Doesn't this integration capture spans by default? Or is that only for incoming HTTP requests? If so, should clarify above in line 21. 

The wording here, "If set to true" seems to imply that this is set to false by default. Is that a correct assumption?

Reply:
it is indeed opt-in today - this changes in v8, but for now if you want to have spans you have to set this to true.

Reply:
Ah gotcha, I made a suggestion above to clarify this! We should update the default behavior after v8 is released.